Include links to the modern homes of the windows modules (#77220)
ansible/ansible: docs/docsite/rst/user_guide/windows_faq.rst

* template (also: win_template)
* wait_for_connection

> Note that as Ansible moves towards Collections more Windows modules have been added and they have migrated into several collections namespaces: `ansible.windows <https://docs.ansible.com/ansible/latest/collections/ansible/windows/index.html>`_, `community.windows <https://docs.ansible.com/ansible/latest/collections/community/windows/index.html>`_, and `chocolatey <https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/index.html#plugins-in-chocolatey-chocolatey>`_.

Reviewer: ```suggestion
Note that as Ansible moves towards Collections more Windows modules have been added and they have migrated into several collections namespaces: `ansible.windows <https://docs.ansible.com/ansible/latest/collections/ansible/windows/index.html>`_, `community.windows <https://docs.ansible.com/ansible/latest/collections/community/windows/index.html>`_, and `chocolatey <https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/index.html>`_.
```

Reviewer 2: I'm going to resolve this particular suggested edits since the links should the :ref: instead. I'll post suggested edits for those next.

> Note that as Ansible moves towards Collections more Windows modules have been added and they have migrated into several collections namespaces: `ansible.windows <https://docs.ansible.com/ansible/latest/collections/ansible/windows/index.html>`_, `community.windows <https://docs.ansible.com/ansible/latest/collections/community/windows/index.html>`_, and `chocolatey <https://docs.ansible.com/ansible/latest/collections/chocolatey/chocolatey/index.html#plugins-in-chocolatey-chocolatey>`_.

Reviewer 2: ```suggestion
Ansible Windows modules exist in the :ref:`plugins_in_ansible.windows`, :ref:`plugins_in_community.windows`, and :ref:`plugins_in_chocolatey.chocolatey` collections.
```
